tests/helpers/perf_run_helper.rb

```diff
@@ -238,15 +238,22 @@ def puppet_infrastructure_tune_current
   #   restart_puppetserver
   #
   def restart_puppetserver
-    puts "Restarting pe-puppetserver service..."
-    puts
+    if ENV["PUPPET_GATLING_SCALE_RESTART_PUPPETSERVER"].eql?("false")
```

Comment:
This doesn't feel right.  The variable is specific to scale testing, and the method is general to any testing.  Right now scale may be the only thing calling this method, but that likely won't always be true.  If I was running some kind of testing other than scale, I wouldn't expect this variable to have an impact but it would.  The conditional should be moved into scale specific methods that call the restsart.

Reply:
Updated